src/main/java/uk/gov/di/ipv/cri/passport/persistence/DataStore.java

```diff
@@ -30,11 +28,13 @@ public DataStore(
         this.dynamoDbEnhancedClient = dynamoDbEnhancedClient;
     }
 
-    public static DynamoDbEnhancedClient getClient() {
+    public static DynamoDbEnhancedClient getClient(URI uri) {
         DynamoDbClient client =
-                new ConfigurationService().isRunningLocally()
-                        ? createLocalDbClient()
-                        : DynamoDbClient.create();
```

Comment:
Using the static create method here is how we would normally connect to the real AWS instance of dynamoDB. With this new URI I think we'll need to set the new env var?

Or maybe this could be slightly refactored to use our IS_LOCAL boolean env var to use the localhost default dynamoDB uri?

Reply:
Thanks Danny its a good catch, the necessary env var hasn't been added 🤦 . The rationale is to avoid having different client creation code for local/test vs production which have a tendency to get out of sync and ending up testing something different to prod code. As worked on with you and Dan this has now been refactored so an override can be added, else it'll connect to the default endpoint 👍